pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
+specified ``sdist_directory``. If the ``source_directory`` is already an
```

Comment:
I think the first should be true largely because at this stage the metadata doesn't exist for the calling tool to know that it is ``requests-2.7.13`` and not something else (I briefly considered trying to add metadata like what setuptools *sometimes* does, but I avoided it because it adds another layer of complexity to this PEP and I think we can avoid it).

I can adjust this PR to this effect.